requirements-test.txt

```diff
@@ -6,7 +6,7 @@ mypy-extensions==1.0.0
 packaging==23.2
 pluggy==1.3.0
 psygnal==0.9.5
-pydantic==1.10.13
+pydantic>=2.0.0
```

Comment:
Exact versions are used in this file. And you forgot about updating the version in `setup.cfg`.